pytentiostat/executable.py

```diff
@@ -0,0 +1,20 @@
+# This is what you want to execute
+import json
+from ArduinoBoard import ProgramSetup as PS
+#from DataHandeling import DataHandeling as DH
+
+
+
+if __name__ == '__main__':
```

Comment:
This code should be in ``main.py`` at the bottom.

to make an executable we put the code in a module called something like ``pytentiostat`` in a subdirectory on pytentiostat main directory called ``scripts``.  We only need to deal with that later when we release, but for now, don't make a separature executable file for this code.